src/engraving/infrastructure/skyline.cpp

```diff
@@ -280,7 +280,7 @@ double SkylineLine::top(double startX, double endX)
         }
     }
 
-    if (top == DBL_MAX) {
+    if (top >= DBL_MAX) { // > is not possible, but silences a compiler warning (-Wfloat-equal)
```

Comment:
Let's not make this change (same a few lines below), as it only adds noise. There is a reason that `-Wfloat-equal` is not enabled by default... and I think we should not strive to enable it, and only use it occasionally as a tool to find _actually_ problematic comparisons.